ee/clickhouse/queries/funnels/funnel.py

```diff
@@ -52,9 +52,14 @@ def get_step_counts_query(self):
         max_steps = len(self._filter.entities)
         breakdown_clause = self._get_breakdown_prop()
 
-        return f"""SELECT person_id, max(steps) AS steps {self._get_step_time_avgs(max_steps)} {breakdown_clause} FROM (
-            {steps_per_person_query}
-        ) GROUP BY person_id {breakdown_clause}
+        return f"""
+            SELECT person_id, steps {self._get_step_time_avgs(max_steps)} {breakdown_clause} FROM (
```

Comment:
When some sequence of steps is a substring of itself, this previously led to incorrect results (since you want to average out times to convert of only the most-completed attempts)

Reply:
This wasn't fully clear^

wrote a new test to explain this: `test_funnel_with_multiple_incomplete_tries` - when you do multiple incomplete tries, and multiple complete tries, only take into account the complete tries, since those are the ones that get counted (max(steps))